jekyll/_cci2/server/latest/operator/reference-monitoring-stack.adoc

```diff
@@ -0,0 +1,19 @@
+---
+contentTags:
+  platform:
+    - Server v4.8
+    - Server Admin
+---
+= Server Monitoring Stack Reference Helm Chart
+:page-layout: classic-docs
+:page-liquid:
+:page-description: Learn how to deploy your own monitoring stack using our helm chart as a reference.
+:icons: font
+:toc: macro
+:toc-title:
+
+
+CircleCI server customers want to know what metrics are available for gaining insight into the performance of their builds. While we don't support or provide any tooling to provide observability into the running services and infrastructure CircleCI server operates in, we can share a sample of how observability can be set up using the most common tools via a helm chart package.
```

Comment:
Wonder about who we're talking to here, server admins? adding a suggestion to reframe

Reply:
yes this would be really for server admins. i'll update this